docs/installation/integrations/react.md

```diff
@@ -321,7 +321,7 @@ class App extends Component {
 					} }
 					onChange={ ( event, editor ) => {
 						const data = editor.getData();
-						console.log( { event, editor, data } );
+						console.log( { event, data } );
```

Comment:
Data can we large as well, `editor.getData` can significantly delay typing, I would comment out both lines.

Reply:
I read that @FilipTokarski couldn't replicate this lag. I also tried and failed - I couldn't copy this behavior even though I was typing like a mad lad with additional content. However, I believe it can slow down the editor for somebody, so I remove this.

Reply:
Yeah, let's remove those lines. Unfortunately using `getData()` inside `onChange` is a known case and it is problematic for some users. So it's something that we definitely do not recommend.